src/python/aqueduct_executor/start-function-executor.sh

```diff
@@ -1,16 +1,25 @@
 #!/bin/bash
 JOB_SPEC=$1
 OP_PATH=$(python3 -m aqueduct_executor.operators.function_executor.get_extract_path --spec "$JOB_SPEC")
+EXIT_CODE=$?
+if [ $EXIT_CODE != "0" ]; then exit $(($EXIT_CODE)); fi
+
 python3 -m aqueduct_executor.operators.function_executor.extract_function --spec "$JOB_SPEC"
+EXIT_CODE=$?
+if [ $EXIT_CODE != "0" ]; then exit $(($EXIT_CODE)); fi
```

Comment:
This is a bug fix right? Just to clarify, previously the function-executor would just continue on even if extracting the function failed?

Reply:
yes. Also we need to exit with `$((EXIT_CODE))` since the script will exit with 0 even if the underlining cmd failed